### DISCUSSION TITLE
Connected drive subscription question

### DISCUSSION THREAD

**Discussion by Bananenn** [BimmerGestalt/AAIdrive#284] [Q&A]
Hello! Amazing concept and work going on here!
Small question, it says you need an active  BMW ConnectedDrive subscription, what does that mean exactly?
I have an account and some services booked like teleservices and emergency call. But i do not have BMW online nor remote services. Do I need BMW online for this to work? I haven't messed with the apps and things for a long time but if im not mistaken that's needed to access the apps no?

I appreciate any and all replies Thanks!

**Comment by bogdan-calapod** [comment]
Hey there!

As long as you have the `Apps` toggle in the Bluetooth devices menu (where you select the connection type, alongside `Telephone` and `Bluetooth Audio`) you should be good to go.

**Reply by Bananenn** [reply]
I do not I've got a 2017 BMW F20 so i think I should since the production date but I was more so thinking of doing it via USB. But none of the standard apps show up when i connect via usb

**Reply by hufman** [reply]
After you connect via USB, do you see a new device show up in the Manage External Devices list? That new USB device might have an Apps icon within, which might be clickable.
I have heard previously that even the USB connection required the subscription.

**Reply by Bananenn** [reply]
> After you connect via USB, do you see a new device show up in the Manage External Devices list? That new USB device might have an Apps icon within, which might be clickable.
> I have heard previously that even the USB connection required the subscription.

I will double check with cabel. But if i need a subscription  what is that called bmw online? The discription for bmw subscriptions is not the best :) Thanks again for the help.

Edit: I am unable to find Manage External Devices but I can find connections where only Bluetooth is an option in there i can see all phones connected via BT, im my multimedia section i can pick from usb (just says loading... forever) and my BT devices. I think you have a newer I drive (or older) than i have NBT / NBT Evo ID 4

Edit nr 2:
I remember back when I had the 1 month "1 dollar trial" I think I got the BMW calendar in there but no other apps and only via USB. When i Connected via and old Iphone i got some other more interesting apps. Noteworthy is perhaps that I do not have navigation. Question: even tho i got no other apps than callendar would this still be compatible if i where to get the apps thing back again.

Sorry for all the questions and whatnot... been looking for a solution like this since I got the car. I am very excited and if I get it working would be happy to share what I have learnt & some more pictures and videos. To many questions? just tell me and ill try to figure it out myself. Thanks!

**Reply by dlmarien** [reply]
Interesting. I ran into a similar issue with my wife's 2018 X1. When the car was new, all subscriptions i.e. ConnectedDrive were included for the first three years. Her subscription expired on April 19 2021 and now the "app" portion of the connection is no longer available. In her case, she can no longer use Spotify. Seems  ridiculous that the "app" is dependent on a $200 annual sub just for the app functionality, but here we are! Shame that's the case

**Reply by dlmarien** [reply]
Hello. I bit the bullet and spent the $200CAD renewing my wife's Connected Drive subscription. Interestingly, services were restored as expected, however the "App" option also returned!! So now she can use the smartphone app functionality again for Spotify. So Connected Drive services are a definite requirement for the App function to appear/work, at least in Canada

**Reply by Bananenn** [reply]
> Hello. I bit the bullet and spent the $200CAD renewing my wife's Connected Drive subscription. Interestingly, services were restored as expected, however the "App" option also returned!! So now she can use the smartphone app functionality again for Spotify. So Connected Drive services are a definite requirement for the App function to appear/work, at least in Canada

Cool and glad to hear! Might have to bite that same bullet then... if you hsve the time send some pictures of this app in action i thinl we have the same version of idrive

**Comment by dlmarien** [comment]
Hello. Not sure photos would show anything terribly specific. This is a
2018 X1, fully loaded, but had the Connected Drive subscription expire in
mid-April. Having renewed the subscription restored the functionality that
was previously available, more specifically the appearance of the "App"
option in order to have your Smartphone apps appear in iDrive i.e. Spotify,
Google etc. The renewed subscription also restored the Weather app and the
Real-Time-Traffic-Information functionality. Her vehicle is iDrive v6. We
are using Huf's app (v 1.2.2 - build # I forget, but it's the latest build
#). I'd be happy to snap a few pics, but it would help if you could clarify
exactly what you're expecting to see?

**Reply by Bananenn** [reply]
Ahh ok, I see you have I drive 6 No worries then Thanks for all the replies!

**Comment by dlmarien** [comment]
Not a problem!

Sent Wirelessly

________________________________
